transform_stats_to_metrics.go

```diff
@@ -249,26 +261,26 @@ func bucketsToProtoBuckets(countPerBucket []int64) []*metricspb.DistributionValu
 	return distBuckets
 }
 
-func labelValuesFromTags(tags []tag.Tag) []*metricspb.LabelValue {
-	if len(tags) == 0 {
-		return nil
+func labelValuesFromTags(tags []tag.Tag, keysToColumns map[string]int) ([]*metricspb.LabelValue, error) {
+	// It is imperative that we set the "HasValue" attribute,
+	// in order to distinguish missing a label from the empty string.
+	// https://godoc.org/github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1#LabelValue.HasValue
+
+	labelValues := make([]*metricspb.LabelValue, len(keysToColumns))
+	for i := 0; i < len(labelValues); i++ {
+		labelValues[i] = &metricspb.LabelValue{
```

Comment:
Let's move the comment from line 265 to down here so that it is close to where it is actually used and beyond the set up, so
```go
for i := 0; i < len(labelValues); i++ {
		labelValues[i] = &metricspb.LabelValue{}
}
```

Go guarantees that initialized structs will have their zero values set, that for a string is "", that for a boolean is false.